_posts/2025-05-16-last-ten-years.markdown

```diff
@@ -0,0 +1,137 @@
+---
+layout: post
+title:  "CHERIoT: The last ten years"
+date:   2025-05-16
+categories: cheri history 
+author: David Chisnall
+---
+
+![Peter G Neumann shaking hands with the presenter while receiving the IEEE Security and Privacy Test of Time Award, with some of the other coauthors on stage.  A title slide shows the paper title and complete author list](/images/2025-05-14 CHERI test of time.jpg)
+
+This week, we received an IEEE Security and Privacy Test of Time Award for the 2015 paper [*CHERI: A Hybrid Capability-System Architecture for Scalable Software Compartmentalization*](https://www.cl.cam.ac.uk/research/security/ctsrd/pdfs/201505-oakland2015-cheri-compartmentalization.pdf).
+This seemed like a good opportunity to look at how CHERI has changed from the time the paper was published to today's CHERIoT (I'll leave discussing other CHERI variants to some of the other coauthors - there's a lot of ongoing CHERI work beyond this project!).
+
+For those less familiar with the history, the CHERI project started in 2010 (I joined in 2012) with DARPA asking 'If you could change anything about computing to improve security, what would you do?'.
+CHERI built on experiences with Capsicum and historical capability systems to reimagine memory safety.
+
+The CHERIoT project started at Microsoft in 2019 when we began to think about scaling down some of the server-class ideas that we'd been working on for Azure to microcontrollers.
```

Comment:
It feels like there's quite a big jump here. If you move the sentence below about earlier research up a bit it will be more chronological.